source/api-base.cpp

```diff
@@ -105,7 +105,7 @@ void                                      Plugin::API::InitializeAPIs()
 		// DirectX 11
 		try {
 			s_APIInstances.insert(s_APIInstances.end(), std::make_shared<Direct3D11>());
-		} catch (const std::exception& ex) {
+		} catch (const std::exception& PLOG_VAR(ex)) {
```

Comment:
Which warning does this fix? I see no different compiler output before and after this change.

Reply:
it's coming from the enc-amf-test project:

warning C4101: 'ex': unreferenced local variable